js/functions.js

```diff
@@ -0,0 +1,56 @@
+function checkCommentLength(comment, length) {
+  if (comment.length <= length) {
+    return true;
+  }
+  return false;
+}
+
+checkCommentLength('проверяемая строка', 20);
+checkCommentLength('проверяемая строка', 18);
+checkCommentLength('проверяемая строка', 10);
+
+function checkPalindromm(palindrom) {
+  palindrom = palindrom.toLowerCase().replaceAll(' ', '');
+  for (let i = 0; i < palindrom.length / 2; i++) {
+    if (palindrom.at(i) !== palindrom.at(-1 - i)) {
+      return false;
+    }
+  }
+  return true;
```

Comment:
Предпочитаю для читаемости отделять циклы и if конструкции одной пустой строкой, так же и return
```suggestion
  palindrom = palindrom.toLowerCase().replaceAll(' ', '');
  
  for (let i = 0; i < palindrom.length / 2; i++) {
    if (palindrom.at(i) !== palindrom.at(-1 - i)) {
      return false;
    }
  }
  
  return true;
```